x-pack/platform/test/fixtures/es_archives/cases/signals/default/mappings.json

```diff
@@ -3,13 +3,15 @@
   "value": {
     "aliases": {
       ".alerts-security.alerts-default": {
-        "is_write_index": false
+        "is_write_index": false,
+        "is_hidden": true
       },
-      ".siem-signals-default": {
-        "is_write_index": true
+      "siem-signals-default": {
+        "is_write_index": true,
+        "is_hidden": true
       }
     },
-    "index": ".siem-signals-default-000001",
+    "index": "siem-signals-default-000001",
```

Comment:
Could you elaborate on why the `.` has been removed from the index names in this file?